src/ui/components/DropdownMenu/DropdownMenuItem.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
+import React from 'react';
+import classNames from 'classnames';
+
+import Link from 'amo/components/Link';
+
+
+type Props = {|
+  title: string,
+  to?: string,
+  onClick?: Function,
+  detached?: boolean,
+|};
+
+const DropdownMenuItem = ({ title, to, onClick, detached = false }: Props) => {
+  if (to && onClick) {
+    throw new Error('You cannot set both `to` and `onClick` props');
+  }
+
+  if (!to && !onClick) {
+    return (
+      <li className="DropdownMenuItem DropdownMenuItem-section">
+        {title}
+      </li>
+    );
+  }
+
+  const classnames = classNames('DropdownMenuItem-link', {
+    'DropdownMenuItem-link--detached': detached,
+  });
+
+  if (to) {
+    return (
+      <li className="DropdownMenuItem">
+        <Link className={classnames} to={to}>{title}</Link>
```

Comment:
I would like to avoid spreading `props`. I could have a whitelist of `allowedProps` though.

Reply:
Fair enough, but it would be nice to have some kind of test ensuring new props supposed by `Link` were passed. Dunno if that's possible; failing that a comment might be nice. But yeah a whitelist of allowed props works for me.

Reply:
mmh, maybe I can actually pass `Link` directly.